core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/DeleterService.java

```diff
@@ -82,7 +86,18 @@ public default void bulkDelete(Collection<T> objects) throws IOException {
     result.throwIfInvalid();
     for (T object : managedObjects) {
       beforeDelete(object);
-      getDeletionStore().delete(object, getAuthorizationManager().getCurrentUser());
+      try {
```

Comment:
I think this nested try can be flattened.

Reply:
Flattened. Also added better logging for when this happens except for `User` objects, where this is expected since they're connected to way too many things to bother checking